src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -409,10 +409,26 @@ inline std::istream& operator>>(std::istream& is, SchedulingCoreType& core_type)
 static constexpr Property<SchedulingCoreType> scheduling_core_type{"SCHEDULING_CORE_TYPE"};
 
 /**
- * @brief This property allows hyper threading during inference.
+ * @brief This property allows CPU threads pinning during inference.
  * @ingroup ov_runtime_cpp_prop_api
  *
- * Developer can use this property to use or not use hyper threading during inference. If user does not explicitly set
+ * Developer can use this property to use or not use CPU threads pinning during inference. If user does not explicitly
+ * set value for this property, OpenVINO may choose any desired value based on internal logic.
+ *
+ * The following code is example to use this property.
+ *
+ * @code
+ * ie.set_property(ov::hint::use_cpu_pinning(true));
```

Comment:
BTW, all other options are called `enable_` something:
```c++
enable_startup_fallback
enable_execution_fallback
enable_mmap
enable_hyper_threading
enable_profiling
enable_loop_unrolling
```
I proper to have an alignment here

Reply:
> BTW, all other options are called `enable_` something:
> 
> ```c++
> enable_startup_fallback
> enable_execution_fallback
> enable_mmap
> enable_hyper_threading
> enable_profiling
> enable_loop_unrolling
> ```
> 
> I propose having an alignment here

Ok. Will create follow up PR to update use_cpu_pinning and use_hyper_threading

Reply:
https://github.com/openvinotoolkit/openvino/pull/16723